src/controllers/objecting.entity.name.controller.ts

```diff
@@ -8,5 +11,16 @@ import { OBJECTIONS_COMPANY_NUMBER } from "../model/page.urls";
  * @param next
  */
 export const post = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
```

Comment:
eslint has thrown an error for this line:
ESLint: Async arrow function has no 'await' expression.(require-await)

Reply:
This might have been missed in a previous review.

Reply:
Removed this and ammended the fuction.